api/api/controllers/decoders_controller.py

```diff
@@ -124,7 +132,7 @@ def get_download_file(pretty: bool = False, wait_for_complete: bool = False, fil
     data = raise_if_exc(loop.run_until_complete(dapi.distribute_function()))
     response = connexion.lifecycle.ConnexionResponse(body=data["message"], mimetype='application/xml')
 
-    return data
+    return response
```

Comment:
According to this change, we should check in integrated tests the 'Content-type' header in the response (it should be 'application/xml').